pennylane_lightning_kokkos/src/simulator/StateVectorKokkos.cpp

```diff
@@ -17,3 +17,19 @@
 // explicit instantiation
 template class Pennylane::StateVectorKokkos<float>;
 template class Pennylane::StateVectorKokkos<double>;
+
+std::string repr_InitArguments(const Kokkos::InitArguments &a) {
```

Comment:
Should there be a docstring on this function that matches the existing Lightning styling?

Reply:
Yes. 
```
/**
 * @brief Returns a printable representation of Kokkos::InitArguments structs.
 */
```

Reply:
Feel free to commit this :), and add the input arguments as a `@param` entry. An example of we use this can be found in the other definitions across the package. This auto-generates a API docstring to HTML for serving the package (see https://docs.pennylane.ai/projects/lightning-kokkos/en/latest/api/library_root.html for example).